### PR TITLE
Update prefix.py

### PR DESCRIPTION
**Pull request by iyZAIRI**

### CODE REVIEW

File: cogs/prefix.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
+import os
 from sqlite3.dbapi2 import connect
 import discord, json, sqlite3
 from discord.ext import commands
 from contextlib import closing
+from dotenv import load_dotenv
+
+load_dotenv()
```
Comment (nnguyen259) [inline cogs/prefix.py]: this is not needed. We don't use .env file for the actual run
- [ ] Remove this

File: cogs/prefix.py [hunk 2]
```diff
@@ -36,9 +51,9 @@ async def add(self, ctx, *, prefixName):
             try:
                 statement = 'INSERT INTO prefixes VALUES (?, ?)'
                 cursor = db.cursor()
-                cursor.execute(statement, (guildId, prefixName))
+                cursor.execute(statement, (guildId, f'{prefixName}'))
```
Comment (nnguyen259) [inline cogs/prefix.py]: Do we really need this?

File: cogs/prefix.py [hunk 3]
```diff
@@ -26,6 +37,10 @@ async def prefix(self, ctx):
 
     @prefix.command()
     async def add(self, ctx, *, prefixName):
+        if not (prefixName.startswith("\"") and prefixName.endswith("\"")):
+            await ctx.send(f'Invalid prefix. Please put the prefix between quotation marks.')
+            return
+        prefixName=prefixName[1:len(prefixName)-1]
```
Comment (nnguyen259) [inline cogs/prefix.py]: don't need len, can just do `[1:-1]` and it should work

**Comment by Ftayri** [comment]
.

**Review by iyZAIRI — COMMENTED** [review]
.

**Review by elee-github — APPROVED** [review]

**Review by nnguyen259 — APPROVED** [review]

**Pull request merged**